src/config.ml

```diff
@@ -1455,7 +1503,7 @@ let parse_next (effect:parser_effect) initial_state : (parser_state, 'err) resul
             | [], name::_extra_devs ->
               Error (Fmt.strf
                        "[dev %S] stanza without required [dev-type]" name)
-            | _ -> Error "multiple conflicting [dev-type] stanzas present"
+            | _ -> Error "multiple (maybe conflicting) [dev-type] stanzas present"
```

Comment:
The code above aims to detect overlapping non-conflicting dev-type/dev things and explain what conflicts, so I think we should consider it a bug if there are scenarios where this happens. Do you have an example?

Reply:
changed it back to conflicting for now